syscall/syscall.c

```diff
@@ -71,6 +71,17 @@ OE_WEAK OE_DEFINE_SYSCALL1(SYS_chdir)
     return oe_chdir(path);
 }
 
+// TODO: Implement this correctly
+OE_WEAK OE_DEFINE_SYSCALL4_M(SYS_clock_nanosleep)
```

Comment:
Do we have a test for this syscall?

Reply:
I have added a subtask to correctly implement and test this syscall - #3933